AUTHORS

```diff
@@ -1036,6 +1036,7 @@ answer newbie questions, and generally made Django that much better:
     Tyson Clugg <tyson@clugg.net>
     Tyson Tate <tyson@fallingbullets.com>
     Unai Zalakain <unai@gisa-elkartea.org>
+    Ülgen Sarıkavak <https://github.com/ulgens>
```

Comment:
As I am not already in the authors file, if you want to give me that honor here, I won't turn it down.

Reply:
@offbyone Of course 🌻 Do you have a link you want me to use or is your Github profile okay?

Reply:
Works for me!

Reply:
Please not in this PR :)

Reply:
@shangxiao Why?

Reply:
Ah I just saw offbyone was the original PR author, ok should be all good to include in this PR 👍

Reply:
@offbyone Done 🌷